### PR TITLE
003 logic logic

### PR DESCRIPTION
**Pull request by eyalgolan**
Completed:
* coding style fixes
* registration logic
* login logic

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,17 +21,22 @@
   },
   "homepage": "https://github.com/teddy4445/lecture_website_app_generator#readme",
   "dependencies": {
+    "@hapi/joi": "^17.1.1",
     "axios": "^0.19.2",
+    "bcryptjs": "^2.4.3",
+    "bootstrap": "^4.5.2",
```
Comment (teddy4445) [inline package.json:27]: I recall @arielfenster said we won't use bootstrap in this project...

File: src/server/src/models/user.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
+const mongoose = require('mongoose');
+
+const Schema = mongoose.Schema;
+
+const userSchema = new Schema({
```
Comment (teddy4445) [inline src/server/src/models/user.js:5]: we store much more regarding a user - this is not part of this branch, just keep in mind

**Review by teddy4445 — APPROVED** [review]
I approve but suggest to check with @arielfenster regarding the "bootstrap" comment

File: src/server/src/models/user.js (hunk 2 above)
Comment (eyalgolan) [reply src/server/src/models/user.js:5]: What do you mean "not part of the branch"? Should I add more fields to the userSchema?

File: package.json (hunk 1 above)
Comment (eyalgolan) [reply package.json]: fixed

**Pull request merged**